src/header/announcement.de.md

```diff
@@ -0,0 +1,5 @@
+---
+template: announcement
+title: Stakedrop Announcement
+---
+Stake ETH to become a tBTC signer! Join for a live stakedrop event on June 8, 2020. <a href="https://www.crowdcast.io/e/keep-stakedrop---live" target="_blank" rel="noopener noreferrer">RSVP Here</a>
```

Comment:
Isn't the point of the `nonDefaultLocales` bit above to avoid us having to create all of these in the repo in English?

Reply:
Ah yeah it's a particularity of the "file" type collections - the the "Announcement" won't appear in the admin for the other languages to edit unless the file exists.